src/components/forgot-password-drawer/ForgotPasswordDrawer.tsx

```diff
@@ -42,7 +42,7 @@ const createStyles = (themeColors: ThemeColors) =>
     },
 
     restoreAccount: {
-      fontWeight: '700',
+      fontWeight: '600',
```

Comment:
We actually have our own <Text> component now for the font weight stuff, so you should be able to just use that instead of hard-coding here https://github.com/AudiusProject/audius-mobile-client/blob/7cd7b684864032ec4b288cb13f87d70705882e1a/src/components/connect-wallets-drawer/ConnectWalletsDrawer.tsx#L58